hasura/migrations/1645304381138_move_guild_dao_addresses/up.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO dao(contract_address, network, guild_id) 
+SELECT moloch_address, 'mainnet', id FROM guild WHERE moloch_address IS NOT NULL;
+
+UPDATE dao SET network = 'xdai' WHERE contract_address IN ('0x30c9aa17fc30e4c23a65680a35b33e8f3b4198a2', '0x5219ffb88175588510e9752A1ecaA3cd217ca783');
```

Comment:
We should ideally be calling this `gnosis-chain` or `gc` instead of `xdai`. But this is a change we have to do in multiple places, so perhaps best tracked in a separate PR. 

https://github.com/MetaFam/TheGame/issues/1170

Reply:
Let's also remember to double check these addresses are the only gnosis chain daos that we have onboarded so far while all of the rest are on mainnet, before we merged this to master.

Reply:
> Let's also remember to double check these addresses are the only gnosis chain daos that we have onboarded so far while all of the rest are on mainnet, before we merged this to master.

I believe I pulled this data from the prod env, but will double check.